src/jestRunner.ts

```diff
@@ -177,7 +177,7 @@ export class JestRunner {
       config.program = `.yarn/releases/${this.config.getYarnPnpCommand}`;
     }
 
-    const standardArgs = this.buildJestArgs(filePath, currentTestName, false);
+    const standardArgs = this.buildJestArgs(filePath, currentTestName, true);
```

Comment:
actually oops, this breaks the given example on windows + powershell and windows + cmd
Reason is this line:
https://github.com/firsttris/vscode-jest-runner/blob/e3f7c76334c248ff426fd078a7634213b4106174/src/util.ts#L61

```cmd
cmd /C "set "NODE_OPTIONS=--require "c:/Program Files/Microsoft VS Code/resources/app/extensions/ms-vscode.js-debug/src/bootloader.js" --inspect-publish-uid=http" && set "VSCODE_INSPECTOR_OPTIONS={"inspectorIpc":"\\\\.\\pipe\\node-cdp.81396-aa5050a0-4.sock","deferredMode":false,"waitForDebugger":"","execPath":"C:\\Users\\user\\scoop\\apps\\nvm\\current\\nodejs\\nodejs\\node.exe","onlyEntrypoint":false,"autoAttachMode":"always","fileCallback":"C:\\Users\\user\\AppData\\Local\\Temp\\node-debug-callback-b17a11136f9e296d"}" && C:\Users\user\scoop\apps\nvm\current\nodejs\nodejs\node.exe node_modules/jest/bin/jest.js """c:/Users/user/git/examples/examples.test.ts""" -t """test with generated (.*?)""" --runInBand "
```

Maybe this arg should be based on `!isWindows()` instead?